app/assets/javascripts/spotlight/application.js

```diff
@@ -24,5 +24,6 @@
 //= require bootstrap-tagsinput
 //= require jquery.serializejson
 //= require clipboard
+//= require dragula.min
```

Comment:
The `dragula.min.js` should probably be in `vendor` (as an external component) or (even better) fully external (as we do with e.g. openseadragon-rails).

Elsewhere in the project, we've been using `Nestable.js`. I'm a little wary of using two different drag + drop libraries. Dragula looks pretty nice, though, so do you think we should investigate swapping out Nestable for it?